README.md

```diff
@@ -40,32 +40,39 @@ Procedure:
 1. Clone the pelux-manifests git repository.
 2. Start vagrant
 ```bash
-vagrant up --provider="docker"
+vagrant up
 ```
 3. Set variables to be used below
 ```bash
-bitbake_image="core-image-pelux"
-yoctoDir="/home/vagrant/pelux_yocto"
+export bitbake_image="core-image-pelux"
+export yoctoDir="/home/vagrant/pelux_yocto"
+export manifest="pelux-intel.xml"
 ```
-4. Setup bitbake with correct local.conf and bblayers.conf
+4. Do repo init
```

Comment:
Hm, I mostly wanted this to be rendered as a list, but right now it isn't anyway, I need to check that.